### DISCUSSION TITLE
problem the install pakuge flutter_map  in the vscode and adroid studio

### DISCUSSION THREAD

**Discussion by mohammad6162** [fleaflet/flutter_map#979] [Q&A]
I get the following error when adding a package  flutter_map I do not know where the problem comes from ???

HTTP error 403: Forbidden
package:pub/src/http.dart 234:5            _ThrowingClient.send
===== asynchronous gap ===========================
package:pub/src/http.dart 382:18           _ThrottleClient.send
===== asynchronous gap ===========================
package:pub/src/source/hosted.dart 637:22  BoundHostedSource._download.<fn>
===== asynchronous gap ===========================
package:pub/src/io.dart 799:12             withTempDir
===== asynchronous gap ===========================
package:pub/src/source/hosted.dart 634:5   BoundHostedSource._download
===== asynchronous gap ===========================
package:pub/src/source/hosted.dart 478:7   BoundHostedSource.downloadToSystemCache
===== asynchronous gap ===========================
package:pub/src/entrypoint.dart 451:35     Entrypoint._get.<fn>
===== asynchronous gap ===========================
package:pub/src/entrypoint.dart 446:12     Entrypoint._get
===== asynchronous gap ===========================
package:pub/src/entrypoint.dart 269:7      Entrypoint.acquireDependencies
This is an unexpected error. Please run

    dart pub --trace get --no-precompile

and include the logs in an issue on https://github.com/dart-lang/pub/issues/new
pub get failed (server unavailable) -- attempting retry 43 in 64 seconds...
HTTP error 403: Forbidden
package:pub/src/http.dart 234:5            _ThrowingClient.send
===== asynchronous gap ===========================
package:pub/src/http.dart 382:18           _ThrottleClient.send
===== asynchronous gap ===========================
package:pub/src/source/hosted.dart 637:22  BoundHostedSource._download.<fn>
===== asynchronous gap ===========================
package:pub/src/io.dart 799:12             withTempDir
===== asynchronous gap ===========================
package:pub/src/source/hosted.dart 634:5   BoundHostedSource._download
===== asynchronous gap ===========================
package:pub/src/source/hosted.dart 478:7   BoundHostedSource.downloadToSystemCache
===== asynchronous gap ===========================
package:pub/src/entrypoint.dart 451:35     Entrypoint._get.<fn>
===== asynchronous gap ===========================
package:pub/src/entrypoint.dart 446:12     Entrypoint._get
===== asynchronous gap ===========================
package:pub/src/entrypoint.dart 269:7      Entrypoint.acquireDependencies
This is an unexpected error. Please run

    dart pub --trace get --no-precompile

and include the logs in an issue on https://github.com/dart-lang/pub/issues/new
pub get failed (server unavailable) -- attempting retry 44 in 64 seconds...

**Comment by ibrierley** [comment]
Often this means you are trying to access some url that you don't have access to. For example a tile server that needs a password or key etc.

**Reply by JaffaKetchup** [reply]
@mohammad6162 See HTTP status codes for more information
